### PR TITLE
build and release debian packages using Github Actions.

### PR DESCRIPTION
**Pull request by qkaiser**

### CODE REVIEW

**Comment by qkaiser** [comment]
One open point is whether we want to rename package name with `onekey` at the end so they're not confused with the original.

**Comment by martonilles** [comment]
> One open point is whether we want to rename package name with `onekey` at the end so they're not confused with the original.

I would bump the version at least with a suffix in https://github.com/onekey-sec/e2fsprogs/blob/master/debian/changelog we can use `1.47.0-1.ok1`

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build_amd64:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Debian package (AMD64)
+        uses: dawidd6/action-debian-package@v1.5.0
+        with:
+          os_distribution: bullseye
```
Comment (martonilles) [inline .github/workflows/build.yml]: don't we want to build it on bookworm?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+jobs:
+  build_amd64:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Debian package (AMD64)
+        uses: dawidd6/action-debian-package@v1.4.4
+        with:
+          os_distribution: bullseye
```
Comment (martonilles) [inline .github/workflows/release.yml]: bookworm?

File: .github/workflows/build.yml (hunk 1 above)
Comment (qkaiser) [reply .github/workflows/build.yml]: fixed.

File: .github/workflows/release.yml (hunk 2 above)
Comment (qkaiser) [reply .github/workflows/release.yml]: fixed.

**Comment by qkaiser** [comment]
> > One open point is whether we want to rename package name with `onekey` at the end so they're not confused with the original.
> 
> I would bump the version at least with a suffix in https://github.com/onekey-sec/e2fsprogs/blob/master/debian/changelog we can use `1.47.0-1.ok1`

Looks like we'll need more than just a suffix change:

 ```
 Install built packages
  /usr/bin/docker exec e2fsprogs debi --with-depends
  Reading package lists...
  Building dependency tree...
  Reading state information...
  The following additional packages will be installed:
    sash
  Suggested packages:
    doc-base gpart parted doc-debian
  The following NEW packages will be installed:
    comerr-dev e2fsck-static e2fsck-static-dbgsym e2fsprogs-dbgsym
    e2fsprogs-l10n fuse2fs fuse2fs-dbgsym libcom-err2-dbgsym libext2fs-dev
    libext2fs2-dbgsym libss2-dbgsym logsave-dbgsym sash ss-dev
  The following packages will be DOWNGRADED:
    e2fsprogs libcom-err2 libext2fs2 libss2 logsave
  0 upgraded, 14 newly installed, 5 downgraded, 0 to remove and 0 not upgraded.
  E: Packages were downgraded and -y was used without --allow-downgrades.
  debi: error: apt-get install --reinstall ./e2fsprogs_1.47.0-1.ok1_amd64.changes subprocess returned exit status 100
  Error: The process '/usr/bin/docker' failed with exit code 100
```

**Comment by martonilles** [comment]
I think we forked the wrong repo, official e2fsprogs is at https://git.kernel.org/pub/scm/fs/ext2/e2fsprogs.git

Debain uses the debian/master branch and the latest changelog is ahead of us:
https://git.kernel.org/pub/scm/fs/ext2/e2fsprogs.git/tree/debian/changelog?h=debian/master

Bookworm is is at 1.47.0-2 https://packages.debian.org/bookworm/e2fsprogs

**Pull request closed without merging**